Makefile

```diff
@@ -47,7 +47,7 @@ install_tools_mac:
 	brew install helm
 
 build-cli: install_rustup_target
-ifeq ($(TARGET), armv7-unknown-linux-gnueabihf)
```

Comment:
leverage `install_rust_target` to compute `cross` or `cargo` as @simlay as suggested.  Then we can make this simple